GVRf/Framework/jni/objects/scene_object_jni.cpp

```diff
@@ -176,7 +179,7 @@ Java_org_gearvrf_NativeSceneObject_attachEyePointeeHolder(
         jlong jeye_pointee_holder) {
     SceneObject* scene_object = reinterpret_cast<SceneObject*>(jscene_object);
     EyePointeeHolder* eye_pointee_holder =
-            reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
+    reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
```

Comment:
I think that may be the problem.

Please go to the same place and click Edit, change Tab Policy from "tab only" to "space only". And change the profile name(otherwise you can't save the new profile.) That is currently GVRF's C code style.

Sincerely
-Chao Chen